src/models/game/reducer.js

```diff
@@ -26,60 +27,75 @@ const initialState = {
 const gameReducer = (state = initialState, action) => {
   switch (action.type) {
     case gameStarted.type: {
+      return {
+        ...state,
+        phase: (state.phase = 1),
```

Comment:
Typo?

Reply:
Thought I had to reference state.phase, Fixed.